balancer-js/src/modules/data/pool-gauges/repository.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
```

Comment:
It would be nice if you could specify what you're disabling rather than disabling the whole eslint for this file.

Reply:
Ok, disabled rules are now specified.